src/templates/finops-hub/modules/scripts/IngestionSetup.kql

```diff
@@ -1439,6 +1471,74 @@ CommitmentDiscountUsage_transform_v1_0()
 .create-or-alter function
 with (docstring='All recommendations transformed to FOCUS 1.0.', folder='Recommendations')
 Recommendations_transform_v1_0()
+{
```

Comment:
This looks like you're changing v1_0 and not v1_0r2. Can you confirm that your changes are in the right place? Ideally, we should leave v1_0 untouched.